sympy/holonomic/holonomic.py

```diff
@@ -520,7 +520,7 @@ def _singularics_to_ord(self):
             y0 = []
             for i in range(a):
                 y0.append(S(0))
-            y0 += b
+            y0 += [j * factorial(a + i) for i, j in enumerate(b)]
```

Comment:
Yes I will make it `int(a)` instead of `a`. The condition `a == int(a)` is checked [here](https://github.com/sympy/sympy/pull/11451/files#diff-7fb015c226fce39c3c9d594304ed41f2R519) in the function.